### PR TITLE
Uses string substitution. Small cleanup. Fixes #14

### LINKED ISSUE #14: Use string substitution for logging
This is a follow-up of #8.

In STDOUT you can sometimes see `[object Object]`.
e.g. for https://github.com/ionicc/Github-organization-inviter/blob/5381d04484b94a64d836c6b215ce67827fa58e72/main.js#L34
This could be evaded by using [string substitution](https://developer.mozilla.org/en-US/docs/Web/API/console#Using_string_substitutions) in the log call.

### PR DESCRIPTION
**Pull request by Ryuno-Ki**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -28,41 +28,45 @@ app.use(expressIp().getIpInfoMiddleware);
...
+                    email: req.body.email
+                })
+		.catch((error) => {
+                    console.log('Invitation failed %o', error)
+		});
 	    console.log(userdata)
-            console.log("Invitation sent to " + req.body.handle + " At : " + req.body.email);
-            res.send("Invitation sent to " + req.body.handle + " At : " + req.body.email);
+            console.log('Invitation sent to %s At : %s', req.body.handle, req.body.email);
+            res.send(`Invitation sent to ${req.body.handle} At : ${req.body.email}`);
```
Comment (Ryuno-Ki) [inline main.js]: String substitution does not work with Express, but template strings do!

File: main.js [hunk 2]
```diff
@@ -28,41 +28,45 @@ app.use(expressIp().getIpInfoMiddleware);
...
-                email: req.body.email
-            });
+            var invStatus = client.orgs
+	        .createInvitation({
+                    org: ORGANIZATION,
+                    email: req.body.email
+                })
+		.catch((error) => {
+                    console.log('Invitation failed %o', error)
+		});
```
Comment (Ryuno-Ki) [inline main.js]: This could slightly touch #9 @sbvkrishna
Better off to check before perhaps?

**Comment by Ryuno-Ki** [comment]
I left some comments in the PR code part.
Tested the happy path (user exists, org does not) and sad path (user does not exist).
@sbvkrishna should also take a look at this logic for #9.

File: main.js (hunk 2 above)
Comment (sbvkrishna) [reply main.js]: I have made a pull request for better error handling for Organization part - #21

Comment (Ryuno-Ki) [reply main.js]: Looks cool!
I'll rebase in case your PR gets merged first.

**Comment by ionicc** [comment]
I'll merge #21 first @Ryuno-Ki 😄

**Review by ionicc — APPROVED** [review]
Great work!

**Comment by Ryuno-Ki** [comment]
I was just looking for the last 4 min where my PR went :eyes:

**Pull request merged**